pkg/cmd/generate/run/fake/fake.go

```diff
@@ -0,0 +1,13 @@
+package fake
```

Comment:
Maybe you should move all these codes to an isolate package named `generator` or something else. The package `cmd` should be as thin as possible. 
Besides, the package seems tricky, is it from TDD or something else?

Reply:
Code snippets in the `run` package are only used by `kusion generate` command, so we put all it under the `generate` package, the `fake` package contains fake code runner which is necessary for testing, besides this is not a tricky design, the fake mechanism is wildly used in Kubernetes.